Lib/test/test_httplib.py

```diff
@@ -669,6 +690,30 @@ def __new__(cls, value, phrase, description=''):
                 'The client needs to authenticate to gain network access')
         enum._test_simple_enum(CheckedHTTPStatus, HTTPStatus)
 
+    def test_httpstatus_range(self):
+        """Checks that the statuses are in the 100-599 range"""
+
+        for member in HTTPStatus.__members__.values():
+            self.assertGreaterEqual(member, 100)
+            self.assertLessEqual(member, 599)
+
+    def test_httpstatus_category(self):
+        """Checks that the statuses belong to the standard categories"""
+
+        categories = (
+            ((100, 199), "is_informational"),
+            ((200, 299), "is_success"),
+            ((300, 399), "is_redirection"),
+            ((400, 499), "is_client_error"),
+            ((500, 599), "is_server_error"),
+        )
+        for member in HTTPStatus.__members__.values():
+            for category in categories:
```

Comment:
`for (lower, upper), category in categories:`

then use `lower` instead of `category[0][0]`, `upper` instead of `category[0][1]`, and `category` instead of `category[1]`.

Reply:
Good point, see https://github.com/python/cpython/pull/95453/commits/6b8dfc4aa18ccee52d64e90161136489df76941a